### DISCUSSION TITLE
Can't run NOTHING on Apache Netbeans! PLEASE help

### DISCUSSION THREAD

**Discussion by Kong-Pepper** [apache/netbeans#7145] [General]
### Apache NetBeans version

Apache NetBeans 21

### What happened

I'm a beginner. In school computers I can run them, but on my PC, no matter what I do, nothing runs.  I tried reinstalling Apache Netbeans and the JDK and nothing changed. Nowhere I can find a solution.

### Language / Project Type / NetBeans Component

Java Maven Project using Java Modules

### How to reproduce

I can't say :/

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows

### JDK

JDK 21

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else
[Debug Log.txt](https://github.com/apache/netbeans/files/14500251/Debug.Log.txt)

**Comment by JoachimRohde** [comment]
Just a shot in the dark, seeing the "Error while storing the mojo status" log message: can it be that you installed NetBeans with admin priviliges and run it as a normal user? That would cause some permission problems in the filesystem and would explain the message.

**Reply by mbien** [reply]
yeah, this likely is a file permission problem. It appears as maven can't write to the target directory. It could be the NetBeans process permissions or the project files.

**Comment by Kong-Pepper** [comment]
Thank you for the insight. I eventually gave up due to the demands of my course, switched to Eclipse and it worked immediately. I'll re-install Netbeans and try again ASAP.

**Reply by Chris2011** [reply]
Hey @Kong-Pepper did you get any error when you start running NetBeans? Is the problem while installing or on starting NetBeans?
